### PR TITLE
fix(curriculum): typos in challenge description

### LINKED ISSUE #47738: Typos in challenges description
1. This sentence is missing a word
https://github.com/freeCodeCamp/freeCodeCamp/blob/762945baf260afde16f4fd5191ce50a1dc20e3cc/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa2999ec27ec516655eba6.md?plain=1#L20

---

2. Here the word `gallery` should not have quotes inside the code block
https://github.com/freeCodeCamp/freeCodeCamp/blob/762945baf260afde16f4fd5191ce50a1dc20e3cc/curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md?plain=1#L10

---

3. In this sentence
> The `"from"` array should contain the original names and the `"to"` array

`from` and `to` should not have quotations in the code block
https://github.com/freeCodeCamp/freeCodeCamp/blob/762945baf260afde16f4fd5191ce50a1dc20e3cc/curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md?plain=1#L18

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47738

### CODE REVIEW

**Comment by jeremylt** [comment]
This issue already has a PR open, so I am closing this PR

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ dashedName: state-name-puzzle
 
 Write a function to solve the challenge for a given array of actual U.S. state names, and for fictional state names.
 
-The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
+The function should return an array. Each element should be an object in this form: `{from:[],to:[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
```
Comment (ieahleen) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md:18]: make sure to adjust this line as requested

Comment (nayabatir1) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md]: I misinterpret it.
I thought that I should only remove quotation from Object.
I've done the changes.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ dashedName: state-name-puzzle
 
 Write a function to solve the challenge for a given array of actual U.S. state names, and for fictional state names.
 
-The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
+The function should return an array. Each element should be an object in this form: `{from:[],to:[]}`. The `from` array should contain the original names and the `to` array should contain the resultant names.
```
Comment (ieahleen) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md:18]: Please do not change the object
```suggestion
The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `from` array should contain the original names and the `to` array should contain the resultant names.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa2999ec27ec516655eba6.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ You should use compound assignment to add the string `Right! You win 20 gold!` t
 assert.match(pick.toString(), /text\.innerText\s*\+=\s*('|")Right! You win 20 gold!\1/);
 ```
 
-You should use compound assignment add `20` to the value of `gold`.
+You should use a compound assignment to add `20` to the value of `gold`.
```
Comment (ieahleen) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa2999ec27ec516655eba6.md:20]:
```suggestion
You should use compound assignment to add `20` to the value of `gold`.
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
THank you for your PR, please make the requested changes

**Review by majestic-owl448 — APPROVED** [review]
Thank you for your PR!

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Comment by ShaunSHamilton** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**